docs/manual/docs/cmd/yammer/report/report-activityuserdetail.md

```diff
@@ -0,0 +1,52 @@
+# yammer report activityuserdetail
+
+Gets details about Yammer activity by user.
+
+## Usage
+
+```sh
+yammer report activityuserdetail [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-p, --period [period]`|The length of time over which the report is aggregated. Supported values `D7|D30|D90|D180`
+`-d, --date [date]`|The date for which you would like to view the users who performed any activity. Supported date format is `YYYY-MM-DD`.
```

Comment:
Missing the text `Specify the date or period, but not both.` which is included in the command's help